pandas/tests/dtypes/test_common.py

```diff
@@ -532,16 +532,16 @@ def test_is_complex_dtype():
     (float, np.dtype(float)),
     ('float64', np.dtype('float64')),
     (np.dtype('float64'), np.dtype('float64')),
-    pytest.mark.xfail((str, np.dtype('<U')), ),
+    (str, np.dtype(str)),
```

Comment:
hmm, that looks like a py2/py3 thing. ok then. (you can maybe add a comment for that particular case)